src/main/riddl/Applications/application.riddl

```diff
@@ -1,18 +1,20 @@
 application UserInterface is {
   command EstablishOrganization is {
-    info: MemberContext.EditableInfo briefly "Details about an organization required for it to exist. See OrganizationContext.Organization.Info for details.",
+    info: OrganizationContext.EditableInfo briefly "Details about an organization required for it to exist. See OrganizationContext.Organization.Info for details.",
     tenant: TenantId,
     parent: Parent?,
+    admins: AdminList,
     members: MemberList,
     owners: OwnerList,
     contacts: ContactList,
-    onBehalfOf: MemberContext.MemberId briefly "The Id of the Member who is issuing the command."
+    onBehalfOf: UserContext.UserId briefly "The Id of the User who is issuing the command."
```

Comment:
I'm not sure this even has relevance as a necessary change. The Member has a userId but it also has its own unique Id.  It could be possible to just use the MemberId because we don't care about non-member users.